lib/pavilion/plugins/results/table.py

```diff
@@ -42,7 +42,7 @@ def _check_args(self, delimiter=None, col_num=None, has_header=None,
                     col_names = []):
 
         if len(col_names) is not 0:
-            if len(col_names) is not col_num:
+            if len(col_names) is not int(col_num):
```

Comment:
The is keyword isn't appropriate here. It's a pointer comparison:
```C
if ( &a == &b) ...
```

You see it used with `None` a lot because you're not checking if the value is `None` you're checking to see if the value is a pointer to the singleton `None` object (basically saying `&a == NULL`)

Reply:
Though I should probably point out that the None object is obviously not literally located at NULL.

Reply:
`if len(col_names) != int(col_num)`